server.go

```diff
@@ -297,78 +294,34 @@ func NewFromConfig(conf Config) (ret Server, err error) {
 
 		// configure Datadog as sink
 		if ret.DDTraceAddress != "" {
-			ret.tracerSinks = append(ret.tracerSinks, tracerSink{
-				name:   "Datadog",
-				tracer: nil,
-				flush:  flushSpansDatadog,
-			})
-			log.Info("Configured Datadog trace sink")
-		}
 
-		// configure Lightstep as Sink
-		if ret.traceLightstepAccessToken != "" {
-			var host *url.URL
-			host, err = url.Parse(conf.TraceLightstepCollectorHost)
+			var ddSink spanSink
```

Comment:
Since this function uses named returns, I can't use `:=` in the call below (it has an `err`), so I had to add a var here and the var needs a type?